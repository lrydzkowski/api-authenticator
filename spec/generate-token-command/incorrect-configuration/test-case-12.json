{
  "app - dev": {
    "clientId": "test-client-id",
    "clientSecret": "test-secret",
    "tokenEndpoint": "https://login.microsoftonline.com/test-tenant/oauth2/token",
    "flow": "client_credentials",
    "keyVault": {
      "vaultUrl": "",
      "secretMappings": {
        "clientSecret": "my-secret"
      }
    }
  }
}
