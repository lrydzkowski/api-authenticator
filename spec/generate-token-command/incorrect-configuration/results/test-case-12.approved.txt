The given configuration is incorrect. Validation errors: '{"keyVault":"When 'keyVault' is specified, 'vaultUrl' is required and cannot be empty."}'.